gallery/streamlit_app.py

```diff
@@ -166,6 +169,9 @@ def page_content():
             if buymeacoffee_username:
                 with next(columns):
                     badge("buymeacoffee", name=buymeacoffee_username)
+            if forum_url:
+                with next(columns):
+                    mention(icon="streamlit", url=forum_url, label="Forum post")
```

Comment:
Ooh, good idea. Great to get more people looking at the forum. Will need to go and back-fill a bunch of the forum post links.